Assets/Scripts/ToggleSetActive.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ToggleSetActive : InteractiveObjects
```

Comment:
Summary of the script is nice, please and thank you.

Reply:
Niceties were added, you are most welcome.